procurement.md

```diff
@@ -19,25 +19,27 @@ multiple vendors to [dramatically reduce their delivery timeline from
 years to
 months](https://18f.gsa.gov/2016/11/15/modular-procurement-state-local-government/).
 
-**KEY RECOMMENDATION**: It is more important to attract bids from
-vendors who are experienced at open source development than to attract
-vendors who are experienced at government contracting. Conduct
+**KEY RECOMMENDATION**: Attract bids from
```

Comment:
Maybe "Take explicit steps to attract bids from..."

(Starting with "Attract bids" is formally active voice, but semantically it's kind of not, because it's the attracted party that is actually taking the action.  All we can advise is that the reader take steps to make it more likely that those parties take that action.)